### ISSUE TITLE
When I search on aniwatch no anime is found

### ISSUE THREAD

**Issue by KazukiP00** [justchokingaround/jerry#65]
https://github.com/justchokingaround/jerry/assets/94195634/081d0245-e541-430d-8abf-155d46c210f0

**Comment by justchokingaround** [comment]
yeah aniwatch is broken atm bc of rabbitstream having a new encryption thats painful to deal with
